README.md

```diff
@@ -662,24 +667,18 @@ Now, all you have to do is implement the Viterbi algorithm in the function below
 
 ```python
 def viterbi(x: list[int], theta: HMMParam) -> ArrayLike:
-    """Compute the K x len(x) Viterbi table (in log-space)."""
-    K, pi, T, E = theta
-    N = len(x)
-    V = np.empty((K, N))
-    # FIXME: fill in V
-    pi, T, E = log_each(pi), log_each(T), log_each(E)
-    # If you know more about numpy, you can do this much more efficiently,
-    # but this is the fundamental algorithm, so it is good to know how to
-    # implement it with the basic tools any language has.
-    for k in range(K):
-        V[k,0] = pi[k] + E[k,x[0]]
-    for i in range(1, len(x)):
-        for k in range(K):
-            V[k,i] = E[k,x[i]] + max(
-                V[kk, i-1] + T[kk, k]
-                for kk in range(K)
-            )
-    return V
+	"""Compute the K x len(x) Viterbi table (in log-space)."""
+	K, pi, T, E = theta
+	N = len(x)
+	V = np.empty((K, N))
+	# FIXME: fill in V
+	pi, T, E = log_each(pi), log_each(T), log_each(E)
+	for i in range(K):
+		V[i,0] = pi[i] + E[i, x[0]]
+	for l in range(1, N):
+		for i in range(K):
+			V[i,l] = E[i, x[l]] + np.max(V[:, l-1] + T[:, i])  #det er det her vi ikke fatter, præcist det med max, fordi det han skriver max_{} ved vi ikke helt hovrdan skal tolkes
```

Comment:
Fandt I ud af det, så? For koden ser ok ud.

Reply:
Ja, vi fik bare ikke lige fjernet kommentaren.

Reply:
Det var også mit gæt ;)